### PR TITLE
Socint 42 get case from db

### PR DESCRIPTION
**Pull request by rpcatling**
Changes:
- remove New Address Endpoint code and tests
- remove calls to firestore for the Cached/Skeleton case and replace with calls to the local DB.
- remove most of the "get by latest" code since much of it was comparing with cached case which we no longer have
- remove a few minor unecessary bits from application config

See the JIRA for more information ...

JIRA LINK: https://collaborate2.ons.gov.uk/jira/browse/SOCINT-42

### CODE REVIEW

File: src/main/resources/swagger-current.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ info:
   title: ONS Contact Centre API
   version: "5.11.18-oas3"
```
Comment (philwhiles) [inline src/main/resources/swagger-current.yml:4]: Let's reset the version to 1.0.0 and the title to "ONS SDC Contact Centre API"

**Review by philwhiles — CHANGES_REQUESTED** [review]
minor tweak

File: src/main/resources/swagger-current.yml (hunk 1 above)
Comment (rpcatling) [reply src/main/resources/swagger-current.yml]: Done. Also removed the (old) history, since that would just be confusing.

**Review by PeterBochel — APPROVED** [review]

File: api/swagger-current.yml [hunk 2]
```diff
@@ -1,1342 +0,0 @@
-openapi: 3.0.0
```
[the remark this answers is not in the document]
Comment (rpcatling) [reply api/swagger-current.yml:1]: the swagger exists elsewhere. I'm just deleting the copy . The other one needs to be in the classpath to support /version endpoint so this is the one to delete.

**Pull request merged**